src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -596,15 +577,54 @@ void GcsPlacementGroupScheduler::AcquireBundleResources(
   }
 }
 
+absl::flat_hash_map<scheduling::NodeID, ResourceRequest> ToNodeBundleResourcesMap(
+    const std::shared_ptr<BundleLocations> &bundle_locations) {
+  absl::flat_hash_map<scheduling::NodeID, ResourceRequest> node_bundle_resources_map;
+  for (auto &bundle : *bundle_locations) {
+    auto node_id = scheduling::NodeID(bundle.second.first.Binary());
+    const auto &bundle_spec = *bundle.second.second;
+    auto bundle_resource_request = ResourceMapToResourceRequest(
+        bundle_spec.GetFormattedResources(), /*requires_object_store_memory=*/false);
+    node_bundle_resources_map[node_id] += bundle_resource_request;
+  }
+  return node_bundle_resources_map;
+}
+
+void GcsPlacementGroupScheduler::CommitBundleResources(
```

Comment:
NOTE: This function is used to commit the bundle resources to the node resources in the `cluster_resource_manager`, it will add bundle related resources to the node(s).